### PR TITLE
Feat/purple theme modal

### PR DESCRIPTION
**Pull request by MSiliunas**
### 🔦 Summary
This adds 'purple' theme support to `<HeaderLayout />`

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

**Comment by shlomitc** [comment]
image
@wuwa - do you approve this?

File: src/MessageBox/docs/CustomModal.js [hunk 1]
```diff
@@ -25,6 +28,11 @@ export default () => (
           code={FullScreenModalRaw}
           children={<FullScreenModal />}
         />
+        <CodeExample
```
Comment (shlomitc) [inline src/MessageBox/docs/CustomModal.js]: That's no the right place to set the example, let's create a file similar to this called (Premium Modal):
https://github.com/wix/wix-style-react/blob/master/src/MessageBox/docs/DestructiveAlert.js

then, add a visual test same as this:
https://github.com/wix/wix-style-react/blob/master/src/MessageBox/test/MessageBox.visual.js#L16-L19

File: src/MessageBox/docs/CustomModalExamples/PurpleModal.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
```
Comment (shlomitc) [inline src/MessageBox/docs/CustomModalExamples/PurpleModal.js]: this file can be removed, according to the previous comment

**Comment by MSiliunas** [comment]
@shlomitc Created a separate page with visual test. Should be good to go :)

**Comment by milkyfruit** [comment]
@MSiliunas the premium modal would be part of 9.1 Alert theme: zpl.io/VKP5Q8P

**Comment by shlomitc** [comment]
@MSiliunas - can you please fix according to Domas request? Sorry for the ping-pong

**Comment by MSiliunas** [comment]
I talked to Domas and he said you will be refactoring how everything is sorted. So he said its okay to leave it this way

Mario Siliunas

**Pull request merged**